mkosi

```diff
@@ -85,6 +85,7 @@ GPT_ROOT_ARM           = uuid.UUID("69dad7102ce44e3cb16c21a1d49abed3")
 GPT_ROOT_ARM_64        = uuid.UUID("b921b0451df041c3af444c6f280d3fae")
 GPT_ROOT_IA64          = uuid.UUID("993d8d3df80e4225855a9daf8ed7ea97")
 GPT_ESP                = uuid.UUID("c12a7328f81f11d2ba4b00a0c93ec93b")
+GPT_BIOS               = uuid.UUID("2168614864496e6f744e656564454649")
```

Comment:
hmm, is that a gpt partition type uuid used elsewhere? or is it made up for this?

I'd feel more comfortable if this would at least use the logic described here: https://systemd.io/BOOT_LOADER_SPECIFICATION, i.e. use a $BOOT partition that uses the non-ESP boot id `bc13c2ff-59e6-4262-a352-b275fd6f7172`

Reply:
https://en.wikipedia.org/wiki/GUID_Partition_Table#Partition_type_GUIDs

the scheme being implemented here is roughly following the description at https://wiki.archlinux.org/index.php/GRUB#GUID_Partition_Table_.28GPT.29_specific_instructions

Reply:
hmm, if this is a grub-specific concept, then I think we should rename this to GPT_GRUB_BIOS or so

Reply:
Although it was originally grub-specific, I'm not sure if it still is or if it's used by any other bootloader. Every reference to this partition I've seen refers to it as "BIOS boot partition" not "BIOS Grub partition". Examples: https://en.wikipedia.org/wiki/BIOS_boot_partition and the output from sfdisk we use for partitioning:

```
>>> Created a new GPT disklabel (GUID: 0A2B60E5-6C24-074F-B136-3FAEDEE761D6).
/home/lucas/tmp/.mkosi-37m4cqlt1: Created a new partition 1 of type 'EFI System' and of size 256 MiB.
/home/lucas/tmp/.mkosi-37m4cqlt2: Created a new partition 2 of type 'BIOS boot' and of size 1 MiB.
/home/lucas/tmp/.mkosi-37m4cqlt3: Created a new partition 3 of type 'Linux root (x86-64)' and of size 1 GiB.
```

If you still think it's better as GPT_GRUB_BIOS, I'm ok. Or I can add a comment there about this.